components/developer/openjade/Makefile

```diff
@@ -17,40 +17,41 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= openjade
 COMPONENT_VERSION= 1.3.2
-COMPONENT_REVISION= 5
+COMPONENT_REVISION= 6
 COMPONENT_SUMMARY= OpenJade DSSSL-Engine for SGML documents
 COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH= \
-  sha256:1d2d7996cc94f9b87d0c51cf0e028070ac177c4123ecbfd7ac1cb8d0b7d322d1
-COMPONENT_ARCHIVE_URL= \
-  https://sourceforge.net/projects/openjade/files/openjade/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
-COMPONENT_PROJECT_URL = https://openjade.sourceforge.net/
-COMPONENT_FMRI = developer/documentation-tool/openjade
-COMPONENT_CLASSIFICATION = System/Text Tools
-COMPONENT_LICENSE = MIT
-COMPONENT_LICENSE_FILE = COPYING
-
-# openjade segfaults in libostyle.so when compiled using gcc 10
-GCC_VERSION = 7
+COMPONENT_ARCHIVE_HASH= sha256:1d2d7996cc94f9b87d0c51cf0e028070ac177c4123ecbfd7ac1cb8d0b7d322d1
+COMPONENT_ARCHIVE_URL=	https://sourceforge.net/projects/openjade/files/openjade/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	https://openjade.sourceforge.net/
+COMPONENT_FMRI=			developer/documentation-tool/openjade
+COMPONENT_CLASSIFICATION=	System/Text Tools
+COMPONENT_LICENSE=		MIT
+COMPONENT_LICENSE_FILE=	COPYING
 
 include $(WS_MAKE_RULES)/common.mk
 
-PATH=$(PATH.gnu)
+PATH= $(PATH.gnu)
 					
-COMPONENT_PREP_ACTION = 	( cp $(COMPONENT_DIR)/files/jade_style-sheet.dtd  $(@D)/dsssl &&\
-					cd  $(@D) &&\
-					rm -f aclocal.m4 missing &&\
-					cp config/configure.in . &&\
-					PATH="$(PATH)" libtoolize --force &&\
-					aclocal -I config &&\
-					autoconf --force )
+COMPONENT_PREP_ACTION= ( \
+	cp $(COMPONENT_DIR)/files/jade_style-sheet.dtd  $(@D)/dsssl &&\
+	cd  $(@D) &&\
+	rm -f aclocal.m4 missing &&\
+	cp config/configure.in . &&\
+	PATH="$(PATH)" libtoolize --force &&\
+	aclocal -I config &&\
+	autoconf --force )
+
+
+# openjade segfaults in libostyle.so when compiled using gcc 10 or later
+GCC_VERSION= 7
```

Comment:
This should be before `common.mk` include.  Otherwise `USERLAND_REQUIRED_PACKAGES` won't be correct.

Reply:
Sorry, fixed.